src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js

```diff
@@ -27,24 +33,27 @@ const renderField = ({
 }) => (
   <React.Fragment>
     <input {...input} type={type} />
-    <p>
-      {touched
-        && (error && <span>{error}</span>)}
-    </p>
+    <div>
+      {
+        touched
+        && error
+        && <span>{error}</span>
+      }
+    </div>
   </React.Fragment>
 );
 
-const renderAuctionsId = ({ fields }) => (
+const renderAuctionsId = ({ fields, meta: { error } }) => (
   <React.Fragment>
-    {fields.map((member, index) => (
-      <p key={member}>
+    {fields.map((auction, index) => (
+      <div key={auction}>
```

Comment:
Contents of such mappings often are separate "things".
It's worth considering if it's worth extracting to separate method `fields.map(this.renderAuction)`
Or to separate component
`fields.map((auction, index) => <AuctionField auction={auction} key={auction}>)`

Reply:
Feedback applied at https://github.com/hrodberaht/react-learning/pull/29/files#diff-d6d585d71cc74408e6a090ced0f137c3R47